README.md

```diff
@@ -1,34 +1,72 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/canonical/ubuntu-image)](https://goreportcard.com/report/github.com/canonical/ubuntu-image)
 
+ubuntu-image is a tool used for generating bootable images. You can use it to build Ubuntu images such as:
 
-ubuntu-image is used to build the following Ubuntu images:
+- Snap-based Ubuntu Core images from model assertions
 
-* Snap-based Ubuntu Core images from model assertions
-* Classical preinstalled Ubuntu images using image definition
+- Classical preinstalled Ubuntu images using image definitions
 
-Future versions should be generalized to build more (eventually all) Ubuntu images.
+The future versions of this tool will be more generalized, allowing users to build a wider range of Ubuntu images, including all future releases.
 
+## Getting started
 
-## Requirements
+### Requirements
 
-Ubuntu 18.04 (Bionic Beaver) is the minimum platform requirement, but Ubuntu 20.04 (Focal Fossa) or newer is recommended. All required third party packages are available in the Ubuntu archive.
+* Ubuntu 18.04 (Bionic Beaver) or newer (recommended: Ubuntu 20.04 (Focal Fossa) or newer)
 
+* Ability to install snaps ([SnapStore: ubuntu-image](https://snapcraft.io/ubuntu-image))
+
+### Quickstart
+
+See [Build your first Ubuntu Core image](https://ubuntu.com/core/docs/build-an-image) for instructions on how to build an Ubuntu core image on a **Raspberry Pi**.
+
+### Building images
+
+ubuntu-image offers two basic sub-commands for building snap-based and classical images.
+
+To build a snap-based image with ubuntu-image, you need a [model assertion](https://ubuntu.com/core/docs/reference/assertions/model). A model assertion is a YAML file that describes a particular combination of core, kernel, and gadget snaps, along with other declarations, signed with a digital signature asserting its authenticity. The `ubuntu-image` command only requires the path to this model assertion to build snap-based images.
+
+To build snap-based images with `ubuntu-image`, use the following command:
+
+```
+ubuntu-image snap model.assertion
+```
+
+See [Build your first Ubuntu Core image](https://ubuntu.com/core/docs/build-an-image) for more information on building snap-based images using ubuntu-image. To build an image with custom snaps, see [Build an image with custom snaps](https://ubuntu.com/core/docs/custom-images).
+
+Classical images are built from a local [gadget tree](https://github.com/snapcore/pc-gadget) path. The [gadget tree](https://github.com/snapcore/pc-gadget) is a primed [gadget snap](https://snapcraft.io/docs/the-gadget-snap) containing a `gadget.yaml` file in the meta directory and all the necessary bootloader gadget bits built. You can prepare a [gadget tree](https://github.com/snapcore/pc-gadget) by fetching a specially customised [gadget snap](https://snapcraft.io/docs/the-gadget-snap) source and running `snapcraft prime` on it. The resulting tree will be stored in the prime/ directory.
```

Comment:
I know it comes from [this doc](https://canonical-subiquity.readthedocs-hosted.com/en/latest/reference/ubuntu-image.html) but (you cannot know that @activus-d so no worries!) I really find this confusing to focus a whole paragraph here on something that we do not really want our users to do because building the gadget tree is now done by ubuntu-image if we specify a path to a gadget.yaml file in the image definition YAML. The *most important* input for a classic build is the image definition YAML, currently best documented [here](https://github.com/canonical/ubuntu-image/blob/main/internal/imagedefinition/README.rst) (but we would like that to move). So I think we should focus on that and make it easy to find this doc to easily get started.